src/main/java/com/iheartradio/m3u8/data/Playlist.java

```diff
@@ -1,18 +1,25 @@
 package com.iheartradio.m3u8.data;
 
+import java.util.Collections;
+import java.util.List;
+
 public class Playlist {
     public static final int MIN_COMPATIBILITY_VERSION = 1;
 
     private final MasterPlaylist mMasterPlaylist;
     private final MediaPlaylist mMediaPlaylist;
     private final boolean mIsExtended;
+    private final boolean mIsIframesOnly;
```

Comment:
Similarly, I don't see how this field is being used. Also, I believe EXT-X-I-FRAMES-ONLY is a media tag so the field should belong in the `MediaPlaylist`.